integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/webhook/site/WebhookSiteResource.java

```diff
@@ -21,21 +19,17 @@ public static List<WebhookSiteRequest> requests(WebhookSiteQuerySorting sorting)
                 .jsonPath().getList("data", WebhookSiteRequest.class);
     }
 
-    public static void deleteRequest(final WebhookSiteRequest request) {
+    public static void deleteRequest(final WebhookSiteRequest request, Object webhookID) {
```

Comment:
IMHO `webhookID` can be declared as String, or is there some reason to declare it as Object?

Reply:
Yes, I tried assigning it as a string but that doesn't work in this scenario not sure why, it gives error.

Reply:
What error?

Reply:
In Hooks.java it doesn't accept the method parameter, when it is declared as a string, but when declared as Object it works fine.